src/components/_tests_/DateInput.spec.ts

```diff
@@ -7,8 +7,7 @@ import { DateInput, DateInputProps } from "../DateInput";
 describe("DatePicker", () => {
     const shallowRenderDatePicker = (props: DateInputProps) => shallow(createElement(DateInput, props));
     const defaultProps: DateInputProps = {
-        dateAttribute: "date",
-        onChange: jasmine.createSpy("change"),
+        dateattribute: "date",
         onClick: jasmine.createSpy("click"),
```

Comment:
suggest prop declaration to be done as `onClick: jasmine.any(Function),` then used in individual test where onClick is impremented